servicepulse/redirect.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Redirects
+summary: Describes what message redirects are and how to use them.
+tags:
+- ServicePulse
+related:
+- servicepulse/troubleshooting
+---
+
+## Message Redirects
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed message redirects that have been created to send failed messages to an alternate queue when the original processing endpoint address is no longer available.
+
+INFO: Message redirects are only a feature of ServicePulse/ServiceControl and will not alter the routing for NServiceBus endpoints.
+
+To manage redirects go to Configuration page and select tab called `Retry Redirects`.
+
+![Redirects Tab](images/redirects.png 'width=500')
+
+### Managing redirects
+On the redirect page a list of redirects is being displayed containing following information.
```

Comment:
How about:

This page displays the configured redirects that are currently in effect along with the following information: